### PR TITLE
Databend-test support shell and python scripts

### PR DESCRIPTION
**Pull request by sundy-li**
I hereby agree to the terms of the CLA available at: https://databend.rs/policies/cla/

## Summary

Summary about this PR

Databend-test support shell and python scripts

## Changelog

 
- Improvement 
- Build/Testing/CI
 

## Related Issues

Related #2877 

## Test Plan

Unit Tests

Stateless Tests

### CODE REVIEW

**Comment by BohuTANG** [comment]
/lgtm

It's programmable for tests :+1:

File: tests/suites/0_stateless/00_0000_dummy_select_py.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python3
+
+import os
+import sys
+import signal
+
+import boto3
```
Comment (PsiACE) [inline tests/suites/0_stateless/00_0000_dummy_select_py.py:7]: ModuleNotFoundError: No module named 'boto3'

File: scripts/setup/dev_setup.sh [hunk 2]
```diff
@@ -365,7 +365,7 @@ if [[ "$INSTALL_CODEGEN" == "true" ]]; then
   else
     install_pkg python3 "$PACKAGE_MANAGER"
   fi
-  "${PRE_COMMAND[@]}" python3 -m pip install coscmd PyYAML
+  "${PRE_COMMAND[@]}" python3 -m pip install coscmd PyYAML moto[all]
```
Comment (PsiACE) [inline scripts/setup/dev_setup.sh:368]: also install boto3

**Pull request merged**